src/scss/style.scss

```diff
@@ -1,14 +1,198 @@
+
 body {
-  background-color: pink;
-  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
-  height: 2000px;
+  font-family: serif;
+}
+
+h1 {
+  text-align: center;
+}
+
+section {
+  h2 {
+    font-size: 26px;
+    margin-left: 200px;
+    background: white;
+    padding-left: 6px;
+    border-left: 4px solid #e71212;
+  }
+}
+
+.imageText__content {
+  flex-direction: column;
+  margin: auto auto auto 200px;
+
+  h3 {
+    font-size: 26px;
+  }
+
+  p {
+    font-size: 12px;
+  }
+}
+
+.imageText {
+  background: $section-backgroundcolor;
+
+  &__headline--color {
+    color: #e96262;
+  }
+
+  &__button {
+    color: white;
+    background: black;
+    padding: 8px 12px;
+    margin: auto auto auto 0;
+    border: none;
+    font-size: 10px;
+    font-family: serif;
+  }
+
+  img {
+    width: 50%;
+    max-height: 450px;
+  }
+}
+
+
+//for plain JS
+@keyframes slideDown {
+  from {
+    padding: 0 0 0 200px;
+    opacity: 0;
+  }
+
+  to {
+    padding: 10px 0 120px 200px;
+    opacity: 1;
+  }
+}
+
+@keyframes slideUp {
+  from {
+    padding: 10px 0 120px 200px;
+    opacity: 1;
+  }
+
+  to {
+    padding: 0 0 0 200px;
+    opacity: 0;
+  }
 }
 
-.content {
-  position: absolute;
-  top: 600px;
-  background-color: #fefefe;
-  padding: 50px;
-  margin: 20px;
-  box-shadow: inset 1px 1px 11px 0px black;
+
+.contactUs {
+  background: #2f314d;
+  position: relative;
+  padding: 10px 0 120px 200px;
+
+  p {
+    color: white;
+    background: transparent;
+    border: none;
+    font-family: sans-serif;
+    font-size: 25px;
+    font-weight: 400;
+    margin-top: 0;
+  }
+
+  &:before {
+    content: "";
+    border: 10px solid transparent;
+    border-bottom: #2f314d;
+
+    border-top: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-bottom: 10px solid #2f314d;
+    border-left: 10px solid transparent;
+
+    position: absolute;
+    top: -19px;
+  }
+
+  &--origin {
+    height: 0;
+    padding: 0;
+    visibility: hidden;
+  }
+
+  &--visible {
+    visibility: visible;
+    animation: slideDown 0.5s linear;
+    margin-top: 15px;
+  }
+
+  &--hidden {
+    visibility: visible;
+    animation: slideUp 0.5s linear forwards;
+    margin-top: 15px;
+  }
 }
+
+/*
+//For jQuery
+.contactUs {
+  background: #2f314d;
+  position: relative;
+  padding: 10px 0 120px 200px;
+  margin-top: 15px;
+
+  p {
+    color: white;
+    background: transparent;
+    border: none;
+    font-family: sans-serif;
+    font-size: 25px;
+    font-weight: 400;
+    margin-top: 0;
+  }
+
+  &:before {
+    content: "";
+    border: 10px solid transparent;
+    border-bottom: #2f314d;
+
+    border-top: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-bottom: 10px solid #2f314d;
+    border-left: 10px solid transparent;
+
+    position: absolute;
+    top: -19px;
+  }
+}
+*/
+.imageText--flex {
+  display: flex;
+}
+
+.secondaryConversion--flex {
```

Comment:
Dieser Teil kann innerhalb von .imageText definiert werden 

.imageText {
.secodaryconversion--flex & {...}
}

Alle Formatierungen sollten in einem Container sein.

Reply:
https://codepen.io/VI-Lukas/pen/vQebVK